src/useCreateComponent.tsx

```diff
@@ -22,16 +13,20 @@ export const useCreateComponent = (
 
   React.useLayoutEffect(() => {
     if (wrapperDivRef.current !== null && component === null) {
-      try {
-        (connectInstance as unknown as IConnectJSWithPrivateMethods)
-          // This will be replaced by the npm package version when bundling
-          .setReactSdkAnalytics('_NPM_PACKAGE_VERSION_');
-      } catch (e) {
-        console.log('Error setting React Sdk version with error message: ', e);
-      }
       const newComponent = connectInstance.create(tagName);
       setComponent(newComponent);
       if (newComponent !== null) {
+        try {
```

Comment:
Moving the reactsdk analytics version to the component instead since ConnectInstance might not be fully initialized at this point